### PR TITLE
feat: Implement addtocell command and support for aggregate column families

### PR DESCRIPTION
**Pull request by steveniemitz**
This adds a new command "addtocell" for adding values to aggregate cells, as well as enhancing the column family parser to support creating typed column families.

### CODE REVIEW

File: cbt.go [hunk 1]
```diff
@@ -474,11 +474,12 @@ var commands = []struct {
...
 		Desc: "Create a table",
 		do:   doCreateTable,
-		Usage: "cbt createtable <table-id> [families=<family>:<gcpolicy-expression>,...]\n" +
+		Usage: "cbt createtable <table-id> [families=<family>:<gcpolicy-expression>:<type-expression>,...]\n" +
 			"   [splits=<split-row-key-1>,<split-row-key-2>,...]\n" +
-			"  families     Column families and their associated garbage collection (gc) policies.\n" +
+			"  families     Column families and their associated garbage collection (gc) policies and types.\n" +
 			"               Put gc policies in quotes when they include shell operators && and ||. For gcpolicy,\n" +
 			"               see \"setgcpolicy\".\n" +
+			"               Currently only the type \"intsum\" is supported.\n" +
```
Comment (mutianf) [inline cbt.go:482]: isn't there also byte type?

File: cbtdoc.go [hunk 2]
```diff
@@ -90,40 +93,48 @@ Example:
...
 All values are optional and can be overridden at the command prompt.
 
-# Count rows in a table
 
-Usage:
 
-	cbt count <table-id> [prefix=<row-key-prefix>]
 
-# Create app profile for an instance
+Count rows in a table
```
Comment (mutianf) [inline cbtdoc.go:114]: Why did we remove the `#`?  is it intentional?

File: cbt.go (hunk 1 above)
Comment (steveniemitz) [reply cbt.go:482]: not publicly supported (you can't create a column of type Bytes) so I didn't mention it here or add it to the parser.

File: cbtdoc.go (hunk 2 above)
Comment (steveniemitz) [reply cbtdoc.go:114]: :shrug: thats just what I got when I ran `go generate`.  I'm not sure the logic of any of it :P

**Review by mutianf — APPROVED** [review]

**Pull request merged**